src/Uart/Uart.h

```diff
@@ -0,0 +1,281 @@
+/**@file
+ * This file is part of the Leon3 BSP for the Test Environment.
+ *
+ * @copyright 2022-2023 N7 Space Sp. z o.o.
```

Comment:
Given the current situation, I am not sure there will be 2023. Please don't go that far into the future.

Reply:
This applies to all headers

Reply:
ok